-- Lower the setuptools lower bound to match the version available in the
-- FreeBSD ports tree (devel/py-setuptools currently provides 63.1.0).
-- Upstream pins >=77, but the port policy is to use the tree's setuptools.
-- Also switch PEP 621 `license` to the {text=...} form and drop
-- `license-files` so the in-tree setuptools can read the metadata.

--- pyproject.toml.orig	2026-09-14 14:28:19 UTC
+++ pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=77"]
+requires = ["setuptools>=63"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -11,8 +11,8 @@ keywords = ["base64"]
 readme = "README.rst"
 requires-python = ">=3.9"
 keywords = ["base64"]
-license = "BSD-2-Clause"
-license-files = ["LICENSE", "base64/LICENSE"]
+license = {text = "BSD-2-Clause"}
+# license-files = ["LICENSE", "base64/LICENSE"]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     # Indicate who your project is intended for
@@ -80,7 +80,7 @@ nox = ["nox>=2025.11.12"]
 ]
 lint = ["prek"]
 nox = ["nox>=2025.11.12"]
-build-system = ["setuptools>=77"]
+build-system = ["setuptools>=63"]
 dev = [
     {include-group = "coverage"},
     {include-group = "docs"},
